public/app/features/alerting/unified/components/receivers/GlobalConfigForm.tsx

```diff
@@ -87,6 +87,7 @@ export const GlobalConfigForm = ({ config, alertManagerSourceName }: Props) => {
             option={option}
             error={errors[option.propertyName]}
             pathPrefix={''}
+            secureFields={{}}
```

Comment:
why do you prefer moving this initialisation here instead of having it in the OptionField component default values?

Reply:
It was a source of errors already. In some places `secureFields` property was not properly propagated to child components. Instead, the default value was used which was not correct in that context